### ISSUE TITLE
Accessibility: More Unlabeled Buttons With Talkback

### ISSUE THREAD

**Issue by shortwavesurfer2009** [cake-tech/cake_wallet#895]
Here are some more

On the CakePay screen:
- [x] the buttons for back, account, and filter are unlabeled see screenshots 1, 2, and 3

<details>

![CakePay 1](https://user-images.githubusercontent.com/116814522/234017750-88563a4a-132b-4a49-8934-44d54de51a70.png)

</details>

<details>

![CakePay 2](https://user-images.githubusercontent.com/116814522/234017762-b9e959a0-98a2-4d76-b06b-85e5f3fc3d98.png)

</details>

<details>

![CakePay 3](https://user-images.githubusercontent.com/116814522/234017769-a2158ce1-fc76-45c5-98d2-72a133a4c5d8.png)

</details>

On the send screen
- [x] the buttons for paste from clipboard, scan qr code, and address book are unlabeled. See screenshots 4, 5, and 6

<details>

![send 1](https://user-images.githubusercontent.com/116814522/234019044-b9320cf6-780c-4ffe-9802-d5491d9e60d2.png)

</details>

<details>

![send 2](https://user-images.githubusercontent.com/116814522/234019049-b9a56b46-2dce-4f41-bdf8-d15dcbd1536c.png)

</details>

<details>

![send 3](https://user-images.githubusercontent.com/116814522/234019051-ccce01b3-1ce2-4f38-8940-91fcaa1d1e12.png)

</details>

On the address book screen
- [x] the "+" icon to add a new address is unlabeled. See screenshot 7

<details>

![AddressBook 1](https://user-images.githubusercontent.com/116814522/234019548-235f6400-ee3d-4594-a785-d84da3067b22.png)

</details>

**Comment by SamsungGalaxyPlayer** [comment]
On it, thanks!

**Comment by shortwavesurfer2009** [comment]
This is not fully complete. I will uncheck what is missing.

**Comment by OmarHatem28** [comment]
added in the latest release, please verify it

**Comment by shortwavesurfer2009** [comment]
> added in the latest release, please verify it

In 1.7.1 released 4 days ago all seems fine but the "+" icon +screenshot 7

**Comment by OmarHatem28** [comment]
1.7.2 was released yesterday and it should include the "+" icon

**Comment by shortwavesurfer2009** [comment]
Ah, i will have to wait then as its not on github or fdroid

Sent from Proton Mail mobile

-------- Original Message --------

**Comment by OmarHatem28** [comment]
it should be available now, please let me know if it fixes the issue

**Comment by shortwavesurfer2009** [comment]
Yep, that did the trick. Thanks

Sent from Proton Mail mobile

-------- Original Message --------
